tests/Application/app/Providers/ConcreteProvider.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: alex
+ * Date: 3/02/20
+ * Time: 5:01 PM.
+ */
```

Comment:
Please remove this docblock.

Reply:
Sorry for missing that, phpstorm auto-added it.  Gone now.